cmd/autogen.sh

```diff
@@ -45,6 +45,9 @@ case "$ID" in
 		# patches find their way into the distribution.
 		extra_opts="--libexecdir=/usr/lib/snapd --disable-apparmor"
 		;;
+	solus)
+		extra_opts="--with-snap-mount-dir=/snap --enable-merged-usr --enable-nvidia-arch $@"
```

Comment:
Does anyone actually use this script in packaging? AFAIK, it was originally created for the test harness...